api/src/main/kotlin/edu/wgu/osmt/task/TaskMessageService.kt

```diff
@@ -43,6 +43,7 @@ class TaskMessageService {
         const val publishSkills = "batch-publish-skills"
         const val updateCollectionSkills = "update-collection-skills"
         const val skillsForCollectionCsv = "collection-skills-csv-process"
+        const val removeCollectionSkills = "remove-collection-skills"
```

Comment:
Is this for removing collection skills? Or more generalized to removing a collection? I'm not passive aggressively saying what I think, instead I'm genuinely asking your opinion.

Reply:
you are totally right, it should only be remove collection, it was a copy/paste name routine.

Reply:
changed